classes/Language.php

```diff
@@ -707,14 +707,34 @@ public static function getLangDetails($iso)
     public static function getLocaleByIso($isoCode)
     {
         if (!Validate::isLanguageIsoCode($isoCode)) {
-            throw new Exception(sprintf('The ISO code %s is invalid'));
+            throw new Exception(sprintf('The ISO code %s is invalid', $isoCode));
         }
 
         if ($details = self::getLangDetails($isoCode)) {
             return $details['locale'];
-        } else {
-            return false;
         }
+
+        return false;
+    }
+
+    /**
+     * @param string $locale
+     *
+     * @return string|false|null
```

Comment:
no idea, it was here, maybe if the json file is empty in some part. no matter, i remove it